### ISSUE TITLE
Stream plugin does not exist

### ISSUE THREAD

**Issue by pdewouters** [humanmade/altis-skeleton#11]
After finishing the setup and visiting the admin, I see a notification: `

The plugin stream/stream.php has been deactivated due to an error: Plugin file does not exist.`

**Comment by roborourke** [comment]
@pdewouters can you confirm the version? guessing rc1?

**Comment by roborourke** [comment]
Also was this a chassis or local-server setup?

**Comment by pdewouters** [comment]
yes RC1 and chassis

**Comment by roborourke** [comment]
We've not successfully reproduced this yet, we're just about release RC2 so if you could test again once that's up it'd help. I'll run another test as well.

**Comment by stuartshields** [comment]
@roborourke I can reproduce this locally with chassis. Once you've installed altis v1, just head over to the plugins admin page and you'll get the error reported by @pdewouters.

**Comment by roborourke** [comment]
Yep, I was misunderstanding the issue before - it’s a cosmetic thing caused
by filtering the network activated plugins option so wasn’t a blocker for
release. Will definitely have a fix either in a 1.0.1 release or v2.

Note, I’m going to move this issue to the audit log module repo.

Thanks again for the report and for confirming Stu!
-- 
Robert O'Rourke
Product Engineer
Human Made Limited <https://humanmade.com/>

<https://humanmade.com/enterprise-wordpress-newsletter/?utm_source=email&utm_campaign=email_ro>

*Get the inside track on enterprise WordPress news in your inbox once a
month
<https://humanmade.com/enterprise-wordpress-newsletter/?utm_source=email&utm_campaign=email_ro>.
Make more informed decisions starting today.*

**Comment by joehoyle** [comment]
I've patched upstream in https://github.com/xwp/stream/pull/1003 to be able to make the value filterable. Then I've opened PR https://github.com/humanmade/altis-security/pull/23 to use that if/when it's merged upstream.

**Comment by roborourke** [comment]
@joehoyle looks like maintenance upstream isn't super active so maybe we should publish our fork for the time being
